WDL2CWL_example/varscan/Alignment.cwl

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env cwl-runner
```

Comment:
Does https://github.com/common-workflow-library/bio-cwl-tools/blob/release/bowtie2/bowtie2_build.cwl work for you?

Reply:
ngsap2@ubuntu:~$ sudo docker run -v /var/run/docker.sock:/var/run/docker.sock -v "$PWD":"$PWD" -w="$PWD" commonworkflowlanguage/cwltool /home/ngsap2/tools/bowtie2/bowtie2_build.cwl -c /home/ngsap2/testdata16march/ebola_ref.fasta --bt2_index_base eb
INFO /usr/local/bin/cwltool 2.0.20200107113851
INFO Resolved '/home/ngsap2/tools/bowtie2/bowtie2_build.cwl' to 'file:///home/ngsap2/tools/bowtie2/bowtie2_build.cwl'
ERROR testdata16march/ebola_ref.fasta:1:1: while scanning a block scalar
testdata16march/ebola_ref.fasta:1:2:   expected chomping or indentation indicators, but found 'g'